bridge-token/src/lib.rs

```diff
@@ -56,7 +59,14 @@ impl BridgeToken {
             paused: Mask::default(),
             #[cfg(feature = "migrate_icon")]
             icon: None,
-        }
+        };
+
+        ::near_sdk::env::storage_write(
```

Comment:
This will not work because the contract is deployed by the factory contract and the `owner_set` doesn't allow to set the owner if the predecessor account is not the current account